src/cordova/util.js

```diff
@@ -301,14 +301,23 @@ function isDirectory (dir) {
 
 // Returns the API of the platform contained in `dir`.
 // Potential errors : module isn't found, can't load or doesn't implement the expected interface.
-function getPlatformApiFunction (dir) {
+function getPlatformApiFunction (dir, platform) {
```

Comment:
Apparently we were already passing in the platform name as a second parameter, but not using it 🤷🏼‍♂️